eventsourcing/src/test/java/org/axonframework/eventsourcing/AbstractSnapshotterTest.java

```diff
@@ -141,17 +127,14 @@ void scheduleSnapshotOnlyOnce() {
     @Test
     void scheduleSnapshot_ConcurrencyExceptionIsSilenced() {
         final String aggregateIdentifier = "aggregateIdentifier";
-        doNothing()
-                .doThrow(new ConcurrencyException("Mock"))
-                .when(mockEventStore).storeSnapshot(isA(DomainEventMessage.class));
+        doNothing().doThrow(new ConcurrencyException("Mock"))
+                   .when(mockEventStore).storeSnapshot(isA(DomainEventMessage.class));
         when(mockEventStore.readEvents(aggregateIdentifier))
                 .thenAnswer(invocationOnMock -> DomainEventStream.of(createEvents(2)));
         testSubject.scheduleSnapshot(Object.class, aggregateIdentifier);
 
         testSubject.scheduleSnapshot(Object.class, aggregateIdentifier);
         verify(mockEventStore, times(2)).storeSnapshot(argThat(event(aggregateIdentifier, 1)));
-        verify(logger, never()).warn(anyString());
```

Comment:
That's very good to know. I'll return this behavior in a workable fashion.

Reply:
I ended up taking a slightly different route. 
There's still a custom `Appender` now, but it's defined through the `log4j2.properties` instead.
Took this route since the `Logger` instance (also after casting) in the `AbstractSnapshotter` did not allow the addition of Appenders.